blueprints/extra-machines-resilient-apps.html.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Resilient apps use extra Machines
+layout: docs
+nav: firecracker
+---
+
+Fly Machines are fast-launching VMs; they're the compute of the Fly.io platform. Every Machine is assigned to a single physical host. This helps keep `start` operations consistently fast, but comes at the cost of Machine availability when something goes wrong on a single host.
```

Comment:
I'm almost tempted to leave out the thing about start operations and just say 

```
Every Fly Machine runs on a specific physical host. If that host fails, that Machine simply becomes unavailable; it does not automatically get rescheduled on another host. 

To make your app resilient to single-host failure, create at least two Machines. We have two platform features that will wake up the second Machine when needed.
```

(In place of the paragraph after this too)

Reply:
Thanks. I took some of this and mixed it with that to come up with a better intro. :)